Source/WebKit/UIProcess/Extensions/WebExtensionContext.cpp

```diff
@@ -1562,6 +1562,20 @@ bool WebExtensionContext::purgeMatchedRulesFromBefore(const WallTime& startTime)
     return !m_matchedRules.isEmpty();
 }
 
+void WebExtensionContext::addDeclarativeNetRequestRulesToUserContentController(WebUserContentControllerProxy& controllerProxy)
```

Comment:
Can use a shorter name, since the argument type is unique.

```suggestion
void WebExtensionContext::addDeclarativeNetRequestRules(WebUserContentControllerProxy& controllerProxy)
```